testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/shared/DisabledSharedSessionTestCase.java

```diff
@@ -0,0 +1,126 @@
+/*
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.jboss.as.test.clustering.cluster.web.shared;
```

Comment:
The should go in the single node tests - you don't need multiple nodes to validate this logic.

Reply:
With single node I can validate that session are shared between WARs, but I cannot validate that they are not distributed also to another node.

Reply:
Sure you can.  A distributable web application preserves its web sessions across server restarts, a non-distributable web application does not.